catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -63,16 +66,27 @@ public void onCreate() {
 					.build());
 		}
 
-		Utils.fetchSpeechRecognitionSupportedLanguages(this);
-
 		context = getApplicationContext();
 
 		CatroidKoinHelperKt.start(this, CatroidKoinHelperKt.getMyModules());
 
+		Utils.fetchSpeechRecognitionSupportedLanguages(this);
+
 		defaultSystemLanguage = Locale.getDefault().toLanguageTag();
 
 		googleAnalytics = GoogleAnalytics.getInstance(this);
 		googleAnalytics.setDryRun(BuildConfig.DEBUG);
+
+		setupHms();
+	}
+
+	private void setupHms() {
+		if (AGConnectInstance.getInstance() == null) {
```

Comment:
This is not needed on real devices, but somehow robolectric test will fail without this initialization